### ISSUE TITLE
[Feature Request] - Button colors need to be reworked to enhance contrast

### ISSUE THREAD

**Issue by baiwusanyu-c** [ikun-svelte/ikun-ui#162]
### Clear and concise description of the problem

In the browser contrast test, the color contrast test of the button fails

**Comment by baiwusanyu-c** [comment]
![image](https://github.com/ikun-svelte/ikun-ui/assets/32354856/68af12f4-bea5-4be6-abc7-38cc23ae6daa)
